examples/lua/main.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2018 FU Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Basic lua example application
+ *
+ * @author      Daniel Petry <daniel.petry@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "lauxlib.h"
+#include "lualib.h"
+
+#include "main.lua.h"
+
+int lua_run_script (const char *buffer, size_t buffer_len ){
```

Comment:
please remove space between function name and parentheses, and between `buffer_len` and parentheses.
Also, add a space between the closing parentheses and curly bracket

Reply:
Putting the opening curly bracket on the next line as I see that seems to be the most common way of doing it

Reply:
yes, sorry. Curly bracket here should be in next line